internal/auth/token_test.go

```diff
@@ -46,8 +46,9 @@ func TestTokenRetriever_Refresh(t *testing.T) {
 		client := &http.Client{Transport: transport}
 
 		tr := &TokenRetriever{
-			Secrets: secretsMock,
-			Client:  client,
+			Authenticator: &Authenticator{"https://*.auth0.com/api/v2/", "2iZo3Uczt5LFHacKdM0zzgUO2eG2uDjT", "https://auth0.auth0.com/oauth/device/code", "https://auth0.auth0.com/oauth/token"},
```

Comment:
if we're not using the authenticator values during the tests, I'd suggest putting clearly dummy values here.

Reply:
If I recall correctly the tests required correct URLs but we could probably dummy the id

Reply:
☹️ I didn't read the full test, fixed with dummy vals 👍

Reply:
thanks! I think it is clearer. If we need to update values in a future, this is clearly unrelated now 🙇🏼